pkg/tui/page/chat/chat.go

```diff
@@ -179,7 +180,9 @@ func (p *chatPage) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		return p, cmd
 
 	case editor.SendMsg:
-		cmd := p.processMessage(msg.Content)
+		// Resolve agent commands (if any)
+		content := p.app.ResolveCommand(msg.Content)
```

Comment:
Could the editor do the resolve? It kinda feels like it would be its job and not here, so basically the editor sends a message, and we don't care if the user typed it or if it was extracted from a command